protocol/xmrmaker/instance.go

```diff
@@ -57,15 +60,126 @@ func NewInstance(cfg *Config) (*Instance, error) {
 		go cfg.Network.Advertise()
 	}
 
-	return &Instance{
+	inst := &Instance{
 		backend:        cfg.Backend,
 		dataDir:        cfg.DataDir,
 		walletFile:     cfg.WalletFile,
 		walletPassword: cfg.WalletPassword,
 		offerManager:   om,
 		swapStates:     make(map[types.Hash]*swapState),
 		net:            cfg.Network,
-	}, nil
+	}
+
+	err = inst.checkForOngoingSwaps()
+	if err != nil {
+		return nil, err
+	}
+
+	return inst, nil
+}
+
+func (b *Instance) checkForOngoingSwaps() error {
```

Comment:
I get confused by the letter `b` which normally stands for `backend` in our code. Did you consider calling the receiver here `inst` or maybe calling the struct `Maker` with a receiver named `m`? Just throwing out ideas. We don't have to make them in this PR.

Reply:
yeah, I actually ended up changing this in a follow up PR to this, it was bothering me too lol